dotcom-rendering/scripts/webpack/.swcrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "jsc": {
+    "baseUrl": ".",
+    "minify": {
+      "compress": true
+    },
+    "parser": {
+      "syntax": "typescript",
+      "tsx": true,
+      "decorators": false,
+      "dynamicImport": true
+    },
+    "target": "es2020",
```

Comment:
Is this the same as `@babel/preset-env`?

Reply:
Good question... it's related but not the same transform as `@babel/preset-env`.

`target` specifies the output in terms of ES specification i.e. es2020.

https://swc.rs/docs/configuration/compilation#jsctarget

`preset-env` describes target output in terms of supported browsers so you will get the lowest common denominator of ES syntax given your target browsers.

What's interesting is that DCR currently targets browsers that support `esmodules`:

https://github.com/guardian/dotcom-rendering/blob/1c0f164ea7d4e57a8f9e76850ee8ba2a55d5c1ea/dotcom-rendering/scripts/webpack/webpack.config.browser.js#L83-L89

SWC also supports `preset-env` browserslist style targets so I'll update it to use the `esmodules` equivalent.

https://swc.rs/docs/configuration/supported-browsers

However in the interest of not shipping unnecessary transforms (and hence reducing the bundle size) its worth reviewing if this is an appropriate target to carry forward.